### ISSUE TITLE
Reason: Error upserting embeddings to Pinecone...unsupported protocol scheme ""

### ISSUE THREAD

**Issue by juicetin** [pashpashpash/vault-ai#25]
Error:
```
Failed Files:

    <foo>.pdf
    Reason: Error upserting embeddings to Pinecone: Post "<url>.pinecone.io/vectors/upsert": unsupported protocol scheme ""
```

To reproduce:
Follow the setup instructions as per README on 20/4/23, upload a small file (mine was ~4mb after changing the allowed filesize), then try to ask a question - this error would then show up on the UI

**Comment by DrMax1** [comment]
> Error:
> 
> ```
> Failed Files:
> 
>     <foo>.pdf
>     Reason: Error upserting embeddings to Pinecone: Post "<url>.pinecone.io/vectors/upsert": unsupported protocol scheme ""
> ```
> 
> To reproduce: Follow the setup instructions as per README on 20/4/23, upload a small file (mine was ~4mb after changing the allowed filesize), then try to ask a question - this error would then show up on the UI

Similar Problem here, but I already get the error when trying to upload a file!

**Comment by Jasmin25** [comment]
Same error; I tried with a ~300 kb PDF file size.

**Comment by djm93dev** [comment]
prepend https:// to your pinecone api endpoint address

**Comment by brookesdjb** [comment]
> prepend https:// to your pinecone api endpoint address

Yup, that's solved it for me, I think its not obvious because when you copy the endpoint from pinecone it comes without the https://

**Comment by panjezor** [comment]
this should be added to readme.

**Comment by brookesdjb** [comment]
I think it would be trivial to check the presence of "https://" when loading the endpoint from the env var and prepend it in code if it's missing
